app/models/recipe.rb

```diff
@@ -0,0 +1,6 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+
+  validates :name, presence: true
+
```

Comment:
Thank you.
The `description`, `cooking_time`, and `preparation_time` all have default values in the schema table. There was no need to require their presence since they will be given the default values anyway.